ci/pipelines/docs.yml

```diff
@@ -39,13 +39,13 @@ jobs:
             source:
               repository: ruby
           inputs:
-            - name: omnibus-docs
+            - name: ci-docs
           outputs:
-            - name: omnibus-docs/docs/build
+            - name: ci-docs/docs/build
           caches:
-            - path: omnibus-docs/docs/vendor
+            - path: ci-docs/docs/vendor
           run:
-            dir: omnibus-docs/docs
+            dir: ci-docs/docs
```

Comment:
Do we want to change folder names (etc) in the apps themselves?

Reply:
this references the `ci-docs` above which is an alias for this repo in the context of this pipeline. So think it needs to change no?

Reply:
OK - I don't know much about the tech docs app but I'm guessing if we merge this, then these changes will take effect on the next tech docs merge/deploy?

Reply:
yes this is a tricky change - we need to change the thing (the name of the repo) and references to the thing (contained in the configuration of the  pipelines in concourse). I have changed the the name of the repo. The changes in this PR change the references (including downstream aliases that don't really need to be changed but I have for consistency). The references will only be changed on concourse once `fly set-pipeline ...` is run (either by us manually, or by concourse which in a Inceptionesque twist can use this config to update itself). So in answer to your question, the tech-docs pipeline will be changed when we apply the changes in this branch. But tech docs repo itself should need no changes. Not sure if that's clear at all?

Reply:
I have applied the changes just to the concourse runner pipeline to check it works with the renaming - see here https://cd.gds-reliability.engineering/teams/pay-deploy/pipelines/concourse-runner/jobs/build-and-push/builds/13

Reply:
<img width="800" alt="Screenshot 2021-01-21 at 16 41 58" src="https://user-images.githubusercontent.com/1420504/105382213-be2bcc80-5c07-11eb-9a6d-822623184abb.png">

Reply:
Excellent 👍